scripts/upsert_mgrb_participants.py

```diff
@@ -0,0 +1,177 @@
+import csv
+import logging
+from collections import defaultdict
+from io import StringIO
+from typing import Dict, List
+
+import click
+from google.cloud import storage
+
+from metamist.apis import ParticipantApi
+from metamist.graphql import gql, query
+from metamist.models import (
+    ParticipantUpsert,
+    SampleUpsert,
+)
+
+BUCKET_NAME = 'cpg-mgrb-test-upload'
+
+SAMPLE_QUERY = gql(
+    """
+    query getSampleData($project: String!) {
+        project(name: $project) {
+            id
+        }
+        sample(project: {eq: $project}) {
```

Comment:
I probably would've nested sample under project here, in the spirit of GraphQL and since the two fields are inherently coupled. But I assume the decision here was to simplify parsing later?